lib/mongo/grid/fs_bucket.rb

```diff
@@ -147,6 +147,7 @@ def insert_one(file)
       #   size.
       # @option options [ String ] :write The write concern.
       # @option options [ String ] :read The read preference.
+      # @option options [ Session ] :session The session to use.
```

Comment:
It's not clear to me how the gridfs changes are related to the purpose of this ticket. Did the retryable read assertions uncover bugs in the existing gridfs implementation that these changes fix?

Reply:
The implementation was not compliant with the spec tests added for retryable reads.

Reply:
Sounds good, thanks